### ISSUE TITLE
Importing Collections and Environments into freshly created Workspace not working

### ISSUE THREAD

**Issue by xaviergm** [a85/POSTMan-Chrome-Extension#10334]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

When importing existing collections and environments into a freshly created Workspace, some of the files appear to be imported but later disappear (apparently when backing the data up to Postman cloud).

### Steps To Reproduce

1. Export all files from existing workspace one by one.
2. Create new personal workspace from scratch.
3. Import Files (or Folder, it does not matter).
4. Select all files in folder/list of files.
5. Import them
6. Files appear to be imported and start showing up in the collection and environment lists.
7. The number of collections and environments on the overview tab increase and decrease as files are being processed by Postman.
8. After that, many of the collections and the environments are missing.
9. The collections and environments that are missing are random (i.e. if you repeat the steps, not the same environments and collections will be missing each time you repeat them.)

Importing into scratch pad by signing out from Postman and following the steps above into scratch pad (minus the workspace creation, obviously) works flawlessly.
This leads me to believe the issue is happening during the sync to Postman cloud.

### Environment Information

```markdown
- Operating System: Windows 10
- Platform Type: Native App
- Postman Version: 9.0.3
```

**Comment by jbradley311** [comment]
Can confirm this is also happening to me on Mac

Environment Information
- Operating System: Mac OS Mojave
- Platform Type: Native App
- Postman Version: 9.0.3

**Comment by Swapnil-getpostman** [comment]
Hi @xaviergm , @jbradley311 

Thanks for writing and sharing your feedback.

I tried to replicate it over my Desktop/Native app v8.12.1 and it worked as expected for me. I'm sharing the screen record with this response please do have a look for your reference.

To investigate further, could you share the below details, please?
1. Since how long you're facing this issue? Is it with the latest version of the app? And are you able to replicate it repetitively?
2. Have you tried signing out & sign in back to check if it works as expected? Please make sure to save/sync your unsaved work before you sign out of your account.
3. Are you able to replicate the same over Postman on Web? You can give it a try over the Web version and see if that works as expected - https://go.postman.co/

Please make sure your app sync is working as expected and you are on the latest version of the app. If possible do share your screen record by writing us over the support channel by creating a new request at [this](https://www.postman.com/support/) link.

https://user-images.githubusercontent.com/52168965/135411310-267d53ac-f2d3-45e6-8eba-22f6b7234bad.mp4

**Comment by xaviergm** [comment]
> 1. Since how long you're facing this issue? Is it with the latest version of the app? And are you able to replicate it repetitively?
_**This seems to be happening since this week.
As you can see in the original submission, we had version 9.0.3 yesterday. Today, I upgraded to 9.0.4 and was able to reproduce again.
As part of our formal testing we re-import the environments and collections from our BitBucket server, which is the authoritative source for the configuration to be used in testing. We ran some tests last week and the issue did not seem to appear. However, this week it happens every time to all of our test team**_

> 2. Have you tried signing out & sign in back to check if it works as expected? Please make sure to save/sync your unsaved work before you sign out of your account.
_**I signed out and back in as part of the testing already mentioned in my original submission. I signed out and tested using scratch pad, which worked. I was able to reproduce the issue again after signing back in.**_

> 3. Are you able to replicate the same over Postman on Web? You can give it a try over the Web version and see if that works as expected - https://go.postman.co/
_**I haven't tried to replicate using Postman Web. Will try this next.**_

> Please make sure your app sync is working as expected and you are on the latest version of the app. If possible do share your screen record by writing us over the support channel by creating a new request at [this](https://www.postman.com/support/) link.
_**As mentioned above, I upgraded to 9.0.4 today and was able to reproduce the issue.
I have created a support request (#85730) and attached a video. In the video you can see how the files are processed and the counters go up. All of a sudden, they also start going down.
Out of 6 collections and 24 environments in the folder being imported, only 5 environments are loaded. None of the collections made it.**_

**Comment by postman-staff** [comment]
Hi there,

Thanks for the update and for writing us over the support channel.

Just letting you know that I'm working on your support request but feel free to respond to me on this Github thread if you have any other queries.

via @Swapnil-getpostman | Postman Support

**Comment by xaviergm** [comment]
This seems to be fixed now in version 9.1.1.
Thanks to everyone involved in getting this resolved!
